tutor-integration.md

```
@@ -0,0 +1,381 @@
+
+```nginx
+location /tutor {
+  proxy_pass http://tutor-service:3000;
+  proxy_http_version 1.1;
+  proxy_set_header Host $host;
+  proxy_set_header X-Forwarded-Proto $scheme;
+  proxy_set_header X-Forwarded-Host $host;
+}
+```
+
+3. **Historical — superseded by "Local development" above.** This plan predates the
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)